huntsman/utils/light_control.py

```diff
@@ -0,0 +1,265 @@
+import qhue
+from os import path
+from qhue import Bridge, QhueException, create_new_username
+
+#note, light 5 can be 'uncommented' once installed, or causes problems using currently 
+
+
+#deal with error system - should errors be sent to pocs? how does pocs handle error messages?
```

Comment:
Which error messages are you referring to?

Reply:
Lee commented in the first pr that if a username can't be found and the bridge button needs to be pushed, instead of just giving the user the error it should be sent to pocs? But I can't seem to find the other pr comments after Astrohuntsman/POCS was changed.